docs/overview/changelog.md

```diff
@@ -5,11 +5,12 @@ This page the release history for the Zulip server. See also the
 
 ## Zulip 7.x series
 
-### 7.0 -- unreleased
+### 7.0 -- 2023-05-31
 
-This section is an incomplete draft of the release notes for the next
-major release, and is only updated occasionally. See the [commit
-log][commit-log] for an up-to-date list of all changes.
+This page contains the release history for the Zulip 6.x stable
+release series. See the [current Zulip changelog][latest-changelog]
+for newer release series, or the [commit log][commit-log] for an
+up-to-date list of all changes.
```

Comment:
This paragraph seems like it belongs above under `# Version history`, not here under `### 7.0 -- 2023-05-31`. That’s what we did for 6.0.